html-forms/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <label for="Email">Email:</label>
+      <input id="Email" name="Email" type="text">
+      <div>
+        <label for="Why is coding cool?">Why is coding cool?</label>
+      </div>
+      <select name="Why is coding cool?">
```

Comment:
The value of the `id` attribute on the `select` needs to match the value of the `for` attribute on the `label` it corresponds with.